trunk/source/BA_SiteRepository/Ultimate_E4.17_Java11_Linux.target

```diff
@@ -37,6 +37,7 @@
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
 			<repository location="https://struebli.informatik.uni-freiburg.de/p2-repository-ultimate-dependencies/"/>
+			<unit id="org.yaml.snakeyaml" version="2.2.0"/>
 			<unit id="eo-yaml" version="7.0.8"/>
```

Comment:
Shouldn't we remove all references to eo-yaml now?

Reply:
:+1: Ok. I propose we also switch that to the same library soon-ish (to keep things simple), but I guess it doesn't have to be in this PR.